_includes/scripts.html

```diff
@@ -39,4 +36,64 @@
 </script>
 <!-- Intercom -->
 <script>(function(){var w=window;var ic=w.Intercom;if(typeof ic==="function"){ic('reattach_activator');ic('update',intercomSettings);}else{var d=document;var i=function(){i.c(arguments)};i.q=[];i.c=function(args){i.q.push(args)};w.Intercom=i;function l(){var s=d.createElement('script');s.type='text/javascript';s.async=true;s.src='https://widget.intercom.io/widget/rzcjacgd';var x=d.getElementsByTagName('script')[0];x.parentNode.insertBefore(s,x);}if(w.attachEvent){w.attachEvent('onload',l);}else{w.addEventListener('load',l,false);}}})()</script>
+<script>
+  (function ($) {
+    var cookieInnerHtml = '<div><p>By using this website you agree to our <a href="/cookie-policy/">cookie policy</a></p><button id="cookieModalClose" class="btn btn-primary">OK</button></div>';
+
+    var getCookiebyName = function (name){
+      var pair = document.cookie.match(new RegExp(name + '=([^;]+)'));
+      return !!pair ? pair[1] : null;
+    };
+
+    var setCookie = function (name, value, days) {
+      var expires = '';
+      if (days) {
+        var date = new Date();
+        date.setTime(date.getTime() + (days*24*60*60*1000));
+        expires = '; expires=' + date.toUTCString();
+      }
+      document.cookie = name + '=' + (value || '')  + expires + '; path=/';
+    };
+
+    var initCookie = function () {
+
+      // Don't create cookie notice if already acknowledged
+      if (getCookiebyName('GruntyCookie')) {
+        return;
+      }
+
+      // Create the cookie modal
+      var $cookieModal = $('<div />');
+      $cookieModal.attr('id', 'gruntyCookie');
+      $cookieModal.css('z-index', '2147483647');
+      $cookieModal.html(cookieInnerHtml);
+
+      $(document).on('click', '#cookieModalClose', function (event) {
+        setCookie('GruntyCookie', '1', 365);
+        $cookieModal.hide();
+      });
+
+      //$(document, 'body').append($cookieModal);
+      $('body').append($cookieModal);
+    };
+
+    if (window.hasOwnProperty('Intercom')) {
```

Comment:
I assume this means the cookie code MUST go after the Intercom code. If so, can you add a comment at the top of the cookie code explaining this?